src/clr-angular/data/datagrid/providers/column-toggle-buttons.service.ts

```diff
@@ -7,28 +7,20 @@ import { Injectable, TemplateRef } from '@angular/core';
 import { Observable } from 'rxjs';
 import { Subject } from 'rxjs';
 
-export type ColumnToggleButtons = 'ok' | 'selectAll';
```

Comment:
Oh right, I missed this. We want to still allow for `'ok'` and just ignore it. Not project it, not throw an error, it's just dead HTML on their side. That way it's non-breaking, they can remove it later.

Of course, removing it from the docs like you did in this demo. 👍

Reply:
👍